ego/cli/run.go

```diff
@@ -14,11 +14,17 @@ import (
 
 // Run runs a signed executable in standalone mode.
 func (c *Cli) Run(filename string, args []string) (int, error) {
+	if err := c.checkUnsupportedImports(filename); err != nil {
+		return 1, nil
+	}
 	return launch.RunEnclave(filename, args, c.getEgoHostPath(), c.getEgoEnclavePath(), c.runner)
 }
 
 // Marblerun runs a signed executable as a MarbleRun Marble.
 func (c *Cli) Marblerun(filename string) (int, error) {
+	if err := c.checkUnsupportedImports(filename); err != nil {
+		return 1, nil
```

Comment:
Do you return nil for error on purpose (since the func already prints an error) or should we nevertheless return the err here?